services/core/shared/core/compat/sdk_v5/accounts.js

```diff
@@ -139,6 +139,16 @@ const getAccountsFromCore = async (params) => {
 
 const getAccounts = async params => {
 	const accountsDB = await getAccountsIndex();
+	if (params.sort && params.sort.includes('rank')) {
+		return new Error('Rank based sorting is only supported along delegates accounts');
+	}
```

Comment:
Instead of returning an error, perhaps we should consider redirecting the call to `getDelegates`, to set the user expectations consistent.

Reply:
Let us keep it as is unless explicitly requested.